codeqlAlerts.js

```diff
@@ -19,7 +19,8 @@ const [, , ...args] = process.argv;
 
 //  .number, .created_at, .url, .html_url, .state, .dismissed_by.login, .dismissed_at, .dismissed_reason, .rule.id, .rule.severity, .rule.description, .tool.name, .most_recent_instance.classifications[]]
 function getCodeAlerts(repos) {
-  return Promise.map(repos, ({ name, org }) => {
```

Comment:
I changed this so that if the call to get code alerts for a repo fails, it just catches the error, and it will return an array of any of the successful results.